massa-pos-exports/src/pos_final_state.rs

```diff
@@ -359,4 +386,114 @@ impl PoSFinalState {
         }
         Some(index)
     }
+
+    /// Gets a cycle of the Proof of Stake `cycle_history`. Used only in the bootstrap process.
+    ///
+    /// # Arguments:
+    /// `cursor`: indicates the bootstrap state after the previous payload
+    ///
+    /// # Returns
+    /// The PoS cycle and the updated cursor
+    pub fn get_cycle_history_part(
```

Comment:
The streaming step never go to finished in this function

Reply:
It does